### PR TITLE
Video layout

### PR DESCRIPTION
**Pull request by alexandrewalsh**
New Theater Mode + Several UI fixes when video is displayed.

**Features:**
- Theater Mode, where the entity results drop down below the video.
- Responsive UI of buttons on window resizing
- Responsive UI of video IFrame on window resizing (with attempts to keep the 4:3 aspect ratio)
- Fixed Login Div Animation
- Error Handling via displaying message on searchbar.
- Moved Searchbar to the center
- Moved Theater Mode button to top right corner
- *Important:* Changed method of mocking. Removed mock button as an attempt to make the UI closer to the final product. To mock, add `?mock` to the end of the url on the player page.

Most recent changes can be seen here:
[https://yimingnzhao-dot-step-intern-2020.wl.r.appspot.com/player.html](https://yimingnzhao-dot-step-intern-2020.wl.r.appspot.com/player.html) 

The url to mock captions is here: 
[https://yimingnzhao-dot-step-intern-2020.wl.r.appspot.com/player.html?mock](https://yimingnzhao-dot-step-intern-2020.wl.r.appspot.com/player.html?mock)

### CODE REVIEW

File: src/main/webapp/css/player.css [hunk 1]
```diff
@@ -1,9 +1,82 @@
...
+.container-row {
+    display: flex;
+    flex-direction: row;
+}
+
+/*
+#flex-item-video {
+    width: 50%
+}
+*/
```
Comment (yimingnzhao) [inline src/main/webapp/css/player.css:26]: delete if not using

File: src/main/webapp/css/search-bar.css [hunk 2]
```diff
@@ -212,6 +216,6 @@
 
 @media screen and (max-width: 560px) {
 	.search-wrapper.active .input-holder {
-		width: 200px;
+		width: 175px;
```
Comment (yimingnzhao) [inline src/main/webapp/css/search-bar.css:219]: should we make this relative to screen size?

File: src/main/webapp/js/captions.js [hunk 3]
```diff
@@ -214,7 +216,11 @@ function sendJsonForm(json) {
                     console.log('Total Entities Found: ' + json[key][2]);
                 }
                 else {
-                    output += '<tr><td><span class="word">' + key + ':</span></td> ' + '<td><span class="timestamps">' + epochToTimestamp(JSON.stringify(json[key][0])) + '</span></td></tr>';
+                    output += '<tr><td><span class="word">' + key + ':</span></td>';
+                    for (var time in json[key]) {
+                        output += '<td><span class="timestamps">' + epochToTimestamp(JSON.stringify(json[key][time])) + '</span></td>';
+                    }
```
Comment (yimingnzhao) [inline src/main/webapp/js/captions.js:222]: Should we insert commas to separate timestamps here?

File: src/main/webapp/js/captions.js [hunk 4]
```diff
@@ -229,6 +235,17 @@ function sendJsonForm(json) {
         });
 }
 
+function resizeIFrame() {
```
Comment (yimingnzhao) [inline src/main/webapp/js/captions.js:238]: Add method header and comment the method (I think I wrote this function, so I'll do it)

File: src/main/webapp/player.html [hunk 5]
```diff
@@ -27,7 +27,7 @@
         <script defer src="js/captions.js"></script>
     </head>
     <body>
-        <div class="container">
+        <div id="heading-div" class="container">
             <div>
                 <h1>Lecture Buddy</h1>
                 <div style="position: absolute; top: 10px; right: 10px;"> <!-- move to css -->
```
Comment (yimingnzhao) [inline src/main/webapp/player.html:33]: Move to css?

File: src/main/webapp/css/search-bar.css (hunk 2 above)
Comment (RickyAvina) [reply src/main/webapp/css/search-bar.css:219]: This style is only activated on small screens.

File: src/main/webapp/css/player.css (hunk 1 above)
Comment (yimingnzhao) [reply src/main/webapp/css/player.css]: fixed

File: src/main/webapp/css/search-bar.css (hunk 2 above)
Comment (yimingnzhao) [reply src/main/webapp/css/search-bar.css:219]: ohh, got it!

File: src/main/webapp/js/captions.js (hunk 3 above)
Comment (yimingnzhao) [reply src/main/webapp/js/captions.js]: fixed

**Comment by jiashenwang** [comment]
I am seeing a white banner after login (https://screenshot.googleplex.com/TLnWNQO20u6)

**Pull request merged**